examples/dynamic-configuration/envoy.yaml

```diff
@@ -0,0 +1,53 @@
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: 0.0.0.0
+      port_value: 19000
+
+dynamic_resources:
+  ads_config:
+    api_type: GRPC
+    transport_api_version: V3
+    grpc_services:
+    - envoy_grpc:
+        cluster_name: xds_cluster
+    set_node_on_first_message_only: false
```

Comment:
not sure why this was necessary, but it wouldnt update without it

Reply:
Probably it is related to this: https://github.com/envoyproxy/envoy/issues/9682?

Reply:
cool, ill watch that ticket and update if possible/necessary after